packages/composite-checkout/src/components/radio-button.tsx

```diff
@@ -126,7 +129,7 @@ const Label = styled.label< LabelProps & React.LabelHTMLAttributes< HTMLLabelEle
 		content: '';
 		border: 1px solid ${ ( props ) => props.theme.colors.borderColor };
 		border-radius: 100%;
-		top: 28px;
+		top: 40%;
```

Comment:
These radios dots are rather difficult to work with since they are absolutely positioned before and after the label. It might be worth rebuilding this component in the future so we can properly align the radio > label > and logos with flex or grid.